src/common/SurgeLogger.h

```diff
@@ -0,0 +1,7 @@
+/*
+ * SurgeLogger - quick wrapper to hold reference to and bootstrap the spd logger
+ */
+
+#pragma once
+
```

Comment:
`#pragma once` is a non-standard convention that should not be used. Rather have just plain old ifdef cage.